config/reader.go

```diff
@@ -19,9 +19,10 @@ var (
 )
 
 type Product struct {
-	TFVars  string
-	Product string
-	Module  string
+	TFVars         string
+	Product        string
+	InstallVersion string
```

Comment:
Can we simply name it `Version`?

Reply:
lets keep consistent with jenkins for now, until we refactor all names